aws/resource_aws_fsx_lustre_file_system_test.go

```diff
@@ -382,6 +384,74 @@ func TestAccAWSFsxLustreFileSystem_WeeklyMaintenanceStartTime(t *testing.T) {
 	})
 }
 
+func TestAccAWSFsxLustreFileSystem_DeploymentType(t *testing.T) {
+	var filesystem1, filesystem2 fsx.FileSystem
+	resourceName := "aws_fsx_lustre_file_system.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckFsxLustreFileSystemDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsFsxLustreFileSystemDeploymentType("PERSISTENT_1"),
```

Comment:
i think that it would be best to split this to 2 separate tests  for `PERSISTENT_1`
and just have a plan only test for `SCRATCH_1`
see examples here:
 https://github.com/terraform-providers/terraform-provider-aws/pull/12676/files#diff-17cef5d816ce52cd994aef2105a31a2dR117
https://github.com/terraform-providers/terraform-provider-aws/pull/12676/files#diff-17cef5d816ce52cd994aef2105a31a2dR124

Reply:
Done, I've created 2 separate tests, one for `PERSISTENT_1`, and another for `SCRATCH_2`.

And I've added a plan only test for `SCRATCH_1`.